mlflow/utils/file_utils.py

```diff
@@ -711,3 +711,10 @@ def shutil_copytree_without_file_permissions(src_dir, dst_dir):
             relative_file_path = os.path.relpath(file_path, src_dir)
             abs_file_path = os.path.join(dst_dir, relative_file_path)
             shutil.copyfile(file_path, abs_file_path)
+
+
+def contains_path_separator(path):
```

Comment:
Will this catch escaped path declarations? (i.e. "\\\")?

Reply:
I think it will on windows.

Reply:
but it won't on Linux or Mac